public/js/validations.js

```diff
@@ -8,16 +8,25 @@ function isDate(date) {
 }
 
 function isTime(time) {
-    let pattern = /([01][0-9]|2[0-3]):[0-5][0-9]/
+    let pattern = /([01][0-9]|2[0-3]):[0-5][0-9]/;
     if (pattern.test(time)) {
         return true
     } else {
         return false
     }
 }
 
+function isName(name) {
+    let pattern = /^[A-Za-z][A-Za-z\'\-]+([\ A-Za-z][A-Za-z\'\-]+)*/;
+    if (pattern.test(name)) {
+        return true
+    } else {
+        return false
+    }
```

Comment:
Since if the condition is met on the first it will `return` this `else` is not needed, you can just have:
```js
if (pattern.test(name)) {
    return true;
}

return false;
```